build.sh

```diff
@@ -28,10 +28,13 @@ esac
 io_uring=""
 brickmux=""
 case $JOB_NAME in
-    'gh_regression-test-with-multiplex'|'gh_regression-on-demand-multiplex')
+    'gh_regression-on-demand-multiplex')
         brickmux="--enable-brickmux"
         ;;
-    'gh_smoke-centos7'|'gh_devrpm-el7'|'gh_centos7-regression'|'gh_regression-on-demand-full-run'|'gh_regression-test-with-multiplex'|'gh_regression-test-burn-in')
+    'gh_regression-test-with-multiplex')
```

Comment:
`gh_regression-on-demand-multiplex` should have the same flags. You can remove the first case and add it here.

Reply:
The reason for not adding it in the new case is that it was not duplicated. I will have it added in the new case now.